### PR TITLE
IBX-6315: Edit/Preview embedded items

### PR DESCRIPTION
**Pull request by lucasOsti**
| Question           | Answer
| ------------------ | ------------------
| **JIRA issue**     | [IBX-6315](https://issues.ibexa.co/browse/IBX-6315)
| **Improvement**| yes
| **New feature**    | yes
| **Target version** | 4.6
| **BC breaks**      | no
| **Tests pass**     | yes
| **Doc needed**     | no

Screenshot 2023-09-04 at 15 32 41

**TODO**:
- [ ] Implement feature / fix a bug.
- [ ] Implement tests.
- [ ] Fix new code according to Coding Standards (`$ composer fix-cs`).
- [ ] Ask for Code Review.

### CODE REVIEW

File: src/bundle/Resources/public/js/CKEditor/embed/content-inline/embed-inline-editing.js [hunk 1]
```diff
@@ -6,11 +6,24 @@ import IbexaEmbedContentInlineCommand from './embed-inline-command';
...
     return `<svg class="ibexa-icon ibexa-icon--medium ibexa-icon--secondary">
-        <use xlink:href="${window.ibexa.helpers.icon.getIconPath('embed')}"></use>
-    </svg>
-    <span class="ibexa-embed-content__title">${title}</span>`;
+                <use xlink:href="${window.ibexa.helpers.icon.getIconPath('embed')}"></use>
+            </svg>
+            <span
+                class="ibexa-embed-content__title"
+                data-ibexa-update-content-id="${contentId}"
+                data-ibexa-update-source-data-path="Content.Name">${title}</span>
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/CKEditor/embed/content-inline/embed-inline-editing.js:16]:
```suggestion
                data-ibexa-update-source-data-path="Content.Name"
            >
                ${title}
            </span>
```

File: src/bundle/Resources/public/js/CKEditor/embed/content-inline/embed-inline-ui.js [hunk 2]
```diff
@@ -17,6 +17,10 @@ class IbexaEmbedContentInlineUI extends IbexaEmbedBaseUI {
         return {
             contentId: items[0].ContentInfo.Content._id,
             contentName: items[0].ContentInfo.Content.TranslatedName,
+            locationId: items[0].id,
+            languageCodes: items[0].ContentInfo.Content.CurrentVersion.Version.VersionInfo.VersionTranslationInfo.Language.map(
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/CKEditor/embed/content-inline/embed-inline-ui.js:21]: maybe put `items[0].ContentInfo.Content` into variable? it's also used in line 18 and 19, thanks to this variable you'll also shorten this line

File: src/bundle/Resources/public/js/CKEditor/embed/content/embed-editing.js [hunk 3]
```diff
@@ -6,11 +6,24 @@ import IbexaEmbedContentCommand from './embed-command';
...
     return `<svg class="ibexa-icon ibexa-icon--medium ibexa-icon--secondary">
-        <use xlink:href="${window.ibexa.helpers.icon.getIconPath('embed')}"></use>
-    </svg>
-    <span class="ibexa-embed-content__title">${title}</span>`;
+                <use xlink:href="${window.ibexa.helpers.icon.getIconPath('embed')}"></use>
+            </svg>
+            <span
+                class="ibexa-embed-content__title"
+                data-ibexa-update-content-id="${contentId}"
+                data-ibexa-update-source-data-path="Content.Name">${title}</span>
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/CKEditor/embed/content/embed-editing.js:16]:
```suggestion
                data-ibexa-update-source-data-path="Content.Name"
            >
                ${title}
            </span>
```

File: src/bundle/Resources/public/js/CKEditor/embed/content/embed-ui.js [hunk 4]
```diff
@@ -17,6 +17,10 @@ class IbexaEmbedContentUI extends IbexaEmbedBaseUI {
         return {
             contentId: items[0].ContentInfo.Content._id,
             contentName: items[0].ContentInfo.Content.TranslatedName,
+            locationId: items[0].id,
+            languageCodes: items[0].ContentInfo.Content.CurrentVersion.Version.VersionInfo.VersionTranslationInfo.Language.map(
```
Comment (GrabowskiM) [inline src/bundle/Resources/public/js/CKEditor/embed/content/embed-ui.js:21]: same as above

**Review by GrabowskiM — CHANGES_REQUESTED** [review]

**Review by Gengar-i — APPROVED** [review]

**Review by GrabowskiM — APPROVED** [review]

**Review by dew326 — APPROVED** [review]

**Review by micszo — APPROVED** [review]

**Pull request merged**